compiler/modules/front/ast.bal

```diff
@@ -48,6 +48,13 @@ type AssignStmt record {|
     Expr expr;
 |};
 
+type CompoundAssignStmt record {|
+    LExpr lValue;
+    Expr rexpr;
+    CompoundAssignOp op; 
```

Comment:
It's going to work out better to use the normal binary operators (without the `=`) in the AST i.e.

```
type CompoundAssignStmt record {|
    LExpr lValue;
    Expr expr; // use same field name as AssignStmt
    BinaryArithmeticOp|BinaryBitwiseOp binaryOp;
    err:Position pos;
|};
```

Reply:
I'll change it